core/admin/port_management.py

```
@@ -0,0 +1,449 @@
+        return {
+            "url": f"http://{server_ip}:{port}/metrics",
+            "health": f"http://{server_ip}:{port}/health",
+            "prometheus_target": f"{server_ip}:{port}"
+        }
+    return {}
+
+
+def get_active_ports() -> Dict:
+    """Get all currently active port mappings"""
+    # Clean up expired ports
+    now = datetime.now()
```

Ruff (0.14.10)

`datetime.datetime.now()` called without a `tz` argument

(DTZ005)